lib/content-services/src/lib/aspect-list/aspect-list.component.scss

```diff
@@ -1,14 +1,25 @@
+$dialog-title-height: 100px;
+$dialog-information-height: 44px;
```

Comment:
if there is more or less `information` in the modal, it will not break the screen since it will only be visible at a small screen height like 400 pixels. All modern mobile phones have a height of 600 pixels or more